### ISSUE TITLE
Default aspect ratio wrong

### ISSUE THREAD

**Issue by silid** [RSATom/WebChimera#58]
When playing an mpeg transport stream with 16:9 video the web plugin displays using an odd ratio.

The video plays fine in standalone VLC.

Sample file: https://www.dropbox.com/s/7z5irlkp894gx2q/20150116_090008_000152.ts?dl=0

**Comment by RSATom** [comment]
Yes, It could be due rectangle (i.e. not square) pixel in media. Support of it is not implemented yet.

**Comment by jaruba** [comment]
You can now set default aspect ratio and/or crop with the latest version of WebChimera Player.

Example using your sample:
http://editor.webchimera.org/test_aspect_ratio

Official demo:
http://editor.webchimera.org/aspect_ratio_crop

Documentation:
http://wiki.webchimera.org/index.php?title=.addPlaylist()

**I've found two more plugin bugs related to your sample video:**

- vlcPlayer.time is always 0 during playback

- vlcPlayer.position has a 0.1 delay (starts from 0.1 instead of 0.0 and ends with 1 before playback ends)

**Comment by RSATom** [comment]
Good Job, Jaruba! Thanks!
About bugs you've found - I think it's libvlc's bugs/issues.

**Comment by RSATom** [comment]
I tried to find the way to setup correct aspect ratio - but with no luck yet.

**Comment by jaruba** [comment]
The bugs I've found might be isolated to this specific video source, eighter file corruption, improper file header, improper video cutting tool.. I'd also do a test with another ".ts" file, will try to find one.

**Comment by jaruba** [comment]
I've found another ".ts" file to test with:

    http://dveo.com/downloads/TS-sample-files/San_Diego_Clip.ts

I can confirm that all bugs (Aspect Ratio, vlcPlayer.time and vlcPlayer.position) are related to all ".ts" files.

**Comment by RSATom** [comment]
yes, I'm already met this problem with .ts files on FBVLC

**Comment by RSATom** [comment]
it's defenetely not a bug in video source, it's just has not square pixels...

**Comment by RSATom** [comment]
Problem is, I can't find the way to get correct pixel aspect ratio from libvlc. I have one Idea - will try it tomorrow.
